src/config/mod.rs

```diff
@@ -25,7 +25,7 @@ pub use server::JSON_RPC_ENDPOINT;
 pub use subnet::Subnet;
 
 pub const JSON_RPC_VERSION: &str = "2.0";
-pub const IPC_GATEWAY_ADDR: u64 = 64;
+pub const DEFAULT_IPC_GATEWAY_ADDR: u64 = 64;
```

Comment:
IIRC Henrique is usin this somewhere also, we may need to propagate the change there also. Thanks.